modules/ipi-install-troubleshooting_unable-to-discover-new-bare-metal-hosts-using-the-bmc.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_content-type: PROC
+[id="unable-to-discover-new-bare-metal-hosts-using-the-bmc_{context}"]
+= Unable to discover new bare metal hosts using the BMC
+
+In some cases, the installer will not be able to discover the new bare metal hosts and issue an error, because it cannot mount the remote virtual media share. 
+
+For example: 
+
+[source,json]
+----
```

Comment:
The way the error message looks in the guide is a bit strange. Would it be possible to change? (It shows some strange highlights in red for me)

Reply:
I prettied up the JSON and took out the json highlighting, because it doesn't play well with the rest of the terminal stuff.